core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -1171,6 +1174,26 @@ public Response getWebUIMetrics() {
     }, Configuration.global());
   }
 
+  /**
+   * Gets Web UI heartbeat threads.
+   *
+   * @return the response object
+   */
+  @GET
+  @Path(WEBUI_HEARTBEAT_THREADS)
+  public Response getWebUIHeartbeatThreads() {
+    return RestUtils.call(() -> {
+      WebUIHeartbeatThreads response = new WebUIHeartbeatThreads();
+
+      response.setDebug(Configuration.getBoolean(PropertyKey.DEBUG));
```

Comment:
will the output be different if you set `PropertyKey.DEBUG`? I dont think this is relevant?